proto/decentraland/social_service/v1/social_service.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+package decentraland.social_service.v1;
+
+import "google/protobuf/empty.proto";
+
+// Errors
+
+message InvalidFriendshipAction {}
+
+message InternalServerError {}
+
+// Types
+
+message User { string address = 1; }
+
+message RequestResponse {
+  User user = 1;
+  int64 created_at = 2;
+  optional string message = 3;
+}
+
+message RequestPayload {
+  User user = 1;
+  optional string message = 3;
+}
+
+message Requests {
+  repeated RequestResponse requests = 1;
+}
+
+message AcceptResponse { User user = 1; }
+
+message AcceptPayload { User user = 1; }
+
+message RejectResponse { User user = 1; }
+
+message RejectPayload { User user = 1; }
+
+message DeleteResponse { User user = 1; }
+
+message DeletePayload { User user = 1; }
+
+message CancelResponse { User user = 1; }
+
+message CancelPayload { User user = 1; }
+
+message UpsertFriendshipPayload {
+  oneof action {
+    RequestPayload request = 1;
+    AcceptPayload accept = 2;
+    RejectPayload reject = 4;
+    DeletePayload delete = 5;
+    CancelPayload cancel = 6;
+  }
+}
+
+message MutualFriendsPayload {
+  User user = 1;
+}
+
+message UsersResponse {
+  repeated User users = 1;
+}
+
+message FriendshipRequestsResponse {
+  oneof response {
+    Requests requests = 1;
+    InternalServerError internal_server_error = 2;
+  }
+}
+
+message UpsertFriendshipResponse {
+  message Accepted {}
+  oneof response {
+    Accepted accepted = 1;
+    InvalidFriendshipAction invalid_friendship_action = 2;
+    InternalServerError internal_server_error = 3;
+  }
+}
+
+message FriendshipUpdate {
+  oneof update {
+    RequestResponse request = 1;
+    AcceptResponse accept = 2;
+    RejectResponse reject = 4;
+    DeleteResponse delete = 5;
+    CancelResponse cancel = 6;
+  }
+}
+
+// @deprecated
+service SocialService {  
```

Comment:
I see some differences with the v1 with matrix. I think we should keep the original files as the old client is using it